nim/basolato/server.nimble

```diff
@@ -0,0 +1,23 @@
+# Package
+
+version       = "0.1.0"
+author        = "Anonymous"
+description   = "A new awesome baspolato package"
+license       = "MIT"
+srcDir        = "."
+bin           = @["server"]
+
+backend       = "c"
+
+# Dependencies
+
+requires "nim >= 1.2.4"
+requires "https://github.com/itsumura-h/nim-basolato >= 0.5.1"
+requires "httpbeast >= 0.2.2"
+requires "cligen >= 0.9.41"
+requires "templates >= 0.5"
+requires "bcrypt >= 0.2.1"
+requires "nimAES >= 0.1.2"
+requires "https://github.com/enthus1ast/flatdb >= 0.2.4"
```

Comment:
Is there not a **published** version of `flatdb` ?

Reply:
fixed